frontend/test/__support__/e2e/helpers/e2e-email-helpers.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Make sure you have webmail Docker image running locally:
```

Comment:
Would be great to add this info somewhere in the docs

Reply:
https://github.com/metabase/metabase/blob/master/frontend/test/metabase/scenarios/README.md#running